src/EnergyPlus/HVACMultiSpeedHeatPump.cc

```diff
@@ -4289,47 +3995,40 @@ namespace HVACMultiSpeedHeatPump {
         // SUBROUTINE INFORMATION:
         //       AUTHOR:          Lixing Gu, FSEC
         //       DATE WRITTEN:    June 2007
-        //       MODIFIED         na
-        //       RE-ENGINEERED    na
 
         // PURPOSE OF THIS SUBROUTINE:
         //  This routine will write values to output variables in MSHP
 
-        // METHODOLOGY EMPLOYED:
-
         Real64 TimeStepSysSec = state.dataHVACGlobal->TimeStepSysSec;
 
-        auto &MSHeatPump = state.dataHVACMultiSpdHP->MSHeatPump;
-        auto &MSHeatPumpReport = state.dataHVACMultiSpdHP->MSHeatPumpReport;
+        auto &MSHeatPump = state.dataHVACMultiSpdHP->MSHeatPump(MSHeatPumpNum);
+        auto &MSHeatPumpReport = state.dataHVACMultiSpdHP->MSHeatPumpReport(MSHeatPumpNum);
```

Comment:
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:4302]:(style),[constVariable],Variable 'MSHeatPump' can be declared as reference to const
[src/EnergyPlus/HVACMultiSpeedHeatPump.cc:4303]:(style),[constVariable],Variable 'MSHeatPumpReport' can be declared as reference to const